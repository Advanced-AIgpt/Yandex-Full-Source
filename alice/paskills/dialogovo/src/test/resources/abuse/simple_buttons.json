{
  "response": {
    "text": "блять ты че совсем охуел",
    "tts": "блять ты че совсем охуел",
    "end_session": false,
    "buttons": [
      {
        "title": "блять ты че совсем охуел",
        "payload": {
          "text": "блять ты че совсем охуел"
        },
        "hide": false,
        "url": "https://example.com/"
      },
      {
        "title": "блять ты че совсем охуел",
        "payload": {
          "text": "блять ты че совсем охуел"
        },
        "hide": true,
        "url": "https://example.com/"
      }
    ]
  },
  "version": "1.0"
}
