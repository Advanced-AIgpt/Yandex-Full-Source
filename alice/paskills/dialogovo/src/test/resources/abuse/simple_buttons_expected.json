{
  "response": {
    "text": "<censored> ты че совсем <censored>",
    "tts": "<speaker audio=\"beep_mat_1.opus\"> ты че совсем <speaker audio=\"beep_mat_1.opus\">",
    "end_session": false,
    "buttons": [
      {
        "title": "<censored> ты че совсем <censored>",
        "payload": {
          "text": "блять ты че совсем охуел"
        },
        "hide": false,
        "url": "https://example.com/"
      },
      {
        "title": "<censored> ты че совсем <censored>",
        "payload": {
          "text": "блять ты че совсем охуел"
        },
        "hide": true,
        "url": "https://example.com/"
      }
    ]
  },
  "version": "1.0"
}
